### PR TITLE
THRIFT-4460: PHP Library use PSR-2

### PR DESCRIPTION
**Pull request by RobberPhex**
ref https://issues.apache.org/jira/browse/THRIFT-4460

### CODE REVIEW

**Comment by jeking3** [comment]
Rebase on master as I merged THRIFT-4459.

**Comment by RobberPhex** [comment]
rebased and squashed

File: build/docker/scripts/sca.sh [hunk 1]
```diff
@@ -49,6 +49,10 @@ flake8 --ignore=E501 test/test.py
 flake8 --ignore=E501,E722 test/crossrunner
 flake8 test/features
 
+# PHP code style
+composer install --quiet
+./vendor/bin/phpcs
```
Comment (RobberPhex) [inline build/docker/scripts/sca.sh]: @jeking3 
`phpcs` will check which PHP library match PSR-2.(like `flake8`)

Comment (jeking3) [reply build/docker/scripts/sca.sh]: @RobberPhex this change is causing errors in other builds even though I have rebased on master, see:

https://travis-ci.org/apache/thrift/jobs/341109387

I am going to disable it for now.

**Pull request closed without merging**